internal/cmd/pipeline/command_create.go

```diff
@@ -138,8 +135,23 @@ func (c *command) create(cmd *cobra.Command, _ []string) error {
 	}
 
 	// add Schema Registry Cluster if its provisioned
-	if srCluster != nil {
-		createPipeline.Spec.StreamGovernanceCluster = &streamdesignerv1.ObjectReference{Id: srCluster.GetId()}
+	// check if we need to configure Schema Registry cluster for this pipeline
+	useSchemaRegistry, err := cmd.Flags().GetBool("use-schema-registry")
+	if err != nil {
+		return err
+	}
+
+	if useSchemaRegistry {
+		srCluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(environmentId)
+		if err != nil {
+			// ignore if the SR is not enabled
+			if !strings.Contains(err.Error(), "Schema Registry not enabled") {
```

Comment:
We should have a system test for this since functionality will change if the backend error changes.